tests/test.react-stub-createClass.js

```diff
@@ -0,0 +1,184 @@
+import React, { PropTypes } from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import reactStub from 'react-stub';
+
+
+describe('reactStub createClass', function() {
+
+  it('requires a truthy component', function() {
+    expect(() => reactStub()).to.throw(/realComponent is undefined/);
```

Comment:
I don't think this tests anything different than the other truthy test. Is it needed?